optimum/onnxruntime/utils.py

```diff
@@ -341,3 +351,53 @@ class ORTQuantizableOperator(Enum):
     Resize = "Resize"
     AveragePool = "AveragePool"
     Concat = "Concat"
+
+
+def evaluation_loop(
```

Comment:
should we also update the examples to directly use `evaluation_loop`

https://github.com/huggingface/optimum/blob/f755a58e56597f690be4a0c4bdb549ce0ffd4e03/examples/onnxruntime/optimization/question-answering/run_qa.py#L487